Testing/Temporary/LastTest.log

```diff
@@ -0,0 +1,3 @@
+Start testing: Oct 24 10:58 GMT Summer Time
```

Comment:
Does this file need to be deleted? If so, may want to add also .log to .gitignore.